sdk/react/style/bottomOverlayStyles.json

```diff
@@ -9,5 +9,12 @@
   	"fontSize": 18,
   	"fontFamily": "fontawesome",
   	"color": "white"
+  },
+
+  "cuePoint": {
+    "position": "absolute",
+    "fontSize": 12,
+    "fontFamily": "fontawesome",
```

Comment:
Sorry, there is no specific Alice icon for it. I implemented it by just following the convention with the scrubber.